README.md

```
@@ -1,15 +1,61 @@
-# gardarr
+# Gardarr
+
+<p align="center">
+  <img src="assets/logo.png" alt="Gardarr Logo" width="20%" />
+</p>
+
+![GitHub go.mod Go version](https://img.shields.io/github/go-mod/go-version/gardarr/gardarr?filename=backend%2Fgo.mod)
+[![React](https://img.shields.io/badge/React-%2320232a.svg?logo=react&logoColor=%2361DAFB)](#)
```

markdownlint-cli2 (0.18.1)

No empty links

(MD042, no-empty-links)